Gemfile

```diff
@@ -62,6 +62,9 @@ gem "sshkey",                         "~>1.8.0",   :require => false
 gem "uglifier",                       "~>2.7.1",   :require => false
 gem "websocket-driver",               "~>0.6.3"
 
+# Required by the oVirt 4 provider:
+gem "ovirt-engine-sdk", "=4.0.0.alpha16", :require => false
```

Comment:
It can't be `4.0.x' because the `ovirt-engine-sdk` gem is in pre-release yet. It can be `~> 4.0.0.alpha`, if you prefer that.